Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Animation.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+namespace Know_Your_Nation_Speedy.Models
+{
+    public class Animation
+    {
+        public Animation()
+        {
+            Initialise();
+        }
+
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string FileLocation { get; set; }
+        public string Location { get; set; }
+        public string Description { get; set; }
+        public ICollection<UserAnimation> UserAnimation { get; set; }
```

Comment:
You can give this property a plural name as it is a collection of UserAnimations. Do this for all the `ICollection` fields across all the classes

Reply:
Updated